evm_arithmetization/src/memory/memory_stark.rs

```diff
@@ -214,7 +223,9 @@ impl<F: RichField + Extendable<D>, const D: usize> MemoryStark<F, D> {
 
     /// Generates the `COUNTER`, `RANGE_CHECK` and `FREQUENCIES` columns, given
     /// a trace in column-major form.
-    fn generate_trace_col_major(trace_col_vecs: &mut [Vec<F>]) {
+    /// Also generates the `IS_PRUNED`, `PRUNED_CONTEXT_FREQUENCIES` and
```

Comment:
If this is for the documentation of the function you should also indicate what the function is doing (other that the "also")

Reply:
Hmm, not sure what you mean. The existing comment still applies, but now the function also generates a couple other columns.

Reply:
Ahhh, nevermind. I mistakenly thought the comment started with "Also..."